server/core/src/https/views/profile.rs

```diff
@@ -36,40 +32,28 @@ struct ProfilePartialView {
 pub(crate) async fn view_profile_get(
     State(state): State<ServerState>,
     Extension(kopid): Extension<KOpId>,
-    HxRequest(hx_request): HxRequest,
     VerifiedClientInformation(client_auth_info): VerifiedClientInformation,
-) -> axum::response::Result<Response> {
+) -> Result<ProfileView, WebError> {
     let uat: UserAuthToken = state
         .qe_r_ref
         .handle_whoami_uat(client_auth_info, kopid.eventid)
-        .map_err(|op_err| HtmxError::new(&kopid, op_err))
         .await?;
 
     let time = time::OffsetDateTime::now_utc() + time::Duration::new(60, 0);
 
     let can_rw = uat.purpose_readwrite_active(time);
 
-    let profile_partial_view = ProfilePartialView {
-        menu_active_item: ProfileMenuItems::UserProfile,
-        can_rw,
-        account_name: uat.name().to_string(),
-        display_name: uat.displayname.clone(),
-        legal_name: uat.name().to_string(),
-        email: uat.mail_primary.clone(),
-        posix_enabled: false,
-    };
-    let profile_view = ProfileView {
-        profile_partial: profile_partial_view.clone(),
-    };
-
-    Ok(if hx_request {
-        (
-            HxPushUrl(Uri::from_static("/ui/profile")),
-            HtmlTemplate(profile_partial_view),
-        )
-            .into_response()
-    } else {
-        HtmlTemplate(profile_view).into_response()
+    Ok(ProfileView {
+        profile_partial: ProfilePartialView {
+            menu_active_item: ProfileMenuItems::UserProfile,
+            can_rw,
+            account_name: uat.name().to_string(),
+            display_name: uat.displayname.clone(),
+            // TODO: this should be the legal name
```

Comment:
Given today that it's currently just the uat, I think we rip out anything that isn't uat relevant. Just remove legal name imo.